### PR TITLE
Default Model Details 

### PR DESCRIPTION
**Pull request by bilal-aamer**
Closes https://github.com/anarchy-ai/LLM-VM/issues/126

### CODE REVIEW

**Comment by bilal-aamer** [comment]
@mmirman @cartazio Kindly review

File: README.md [hunk 1]
```diff
@@ -188,9 +188,19 @@ response = client.complete(prompt = 'What is Anarchy?', context = '')
 print(response)
 # Anarchy is a political philosophy that advocates no government...
 ```
+Here are some default model's details:
+| Name | Model_Uri | Model params | Checkpoint file size |
+|---|---|---|---|
+| Neo | EleutherAI/gpt-neo-1.3B | 1.3B | 5.31 GB |
+| Bloom | bigscience/bloom-560m | 1.7B | 1.12 GB |
+| Open Llama | openlm-research/open_llama_3b_v2 | 3B | 6.85 GB |
```
Comment (cartazio) [inline README.md:196]: we dont have this model anymore by default :)

**Comment by cartazio** [comment]
also backtick quote the URIs

**Comment by bilal-aamer** [comment]
Done

File: model_uri_tables.md [hunk 2]
```diff
@@ -11,22 +9,12 @@
 #### Bloom Model  
 
 | URI | Model Params | Checkpoint file size | Is Default?
-|-----|--------------|-----------------------|------------------
```
Comment (cartazio) [inline model_uri_tables.md:14]: why is bloom deleted?

File: model_uri_tables.md [hunk 3]
```diff
@@ -11,22 +9,12 @@
...
-| URI | Model Params | Checkpoint file size | Is Default?
-|-------|-------------|---------------------|------------------
-| openlm-research/open_llama_3b_v2 | 3B  | 6.85 GB | ✅
-| openlm-research/open_llama_7b_v2  | 7B | 13 GB   | ❌
-| openlm-research/open_llama_13b | 13B | 26 GB   | ❌
-
 
+|-----|-------------|-----------------------|------------------
+| bigscience/bloom-560m | 560m | 1.12 GB | ✅
+| bigscience/bloom-1b7/ | 1.7B | 3.4GB   | ❌
```
Comment (cartazio) [inline model_uri_tables.md:15]: nvm, i cant read diffs

**Comment by bilal-aamer** [comment]
Any problem?

**Comment by bilal-aamer** [comment]
@cartazio Could you check on this

**Comment by cartazio** [comment]
Other stuff is priority. Will review when it makes sense

**Comment by bilal-aamer** [comment]
Understandable, however this helps beginners in the space understand which model they can use based on the model params, size, etc.

It's addresses the issue #126 which is open for a month now.

File: README.md
Comment (VictorOdede) [inline README.md:1]: i think line 199 should still be `tables in model_uri_tables`

File: model_uri_tables.md
Comment (VictorOdede) [inline model_uri_tables.md:1]: why did you remove the open-llama table?

**Review by VictorOdede — CHANGES_REQUESTED** [review]

File: README.md
Comment (bilal-aamer) [reply README.md:1]: Was thinking of keeping it either `Model Tables` or `here` keeping it more readable than the `model_uri_tables`

Comment (VictorOdede) [reply README.md:1]: having the actual file name avoids ambiguity

Comment (bilal-aamer) [reply README.md:1]: Sure will change it right away.

Comment (bilal-aamer) [reply README.md:1]: @VictorOdede Done

**Comment by bilal-aamer** [comment]
@VictorOdede Any update on this?

**Comment by bilal-aamer** [comment]
@mmirman @VictorOdede This PR is standing for a while now, kindly review

**Review by VictorOdede — APPROVED** [review]

**Pull request merged**